src/legacy/ui/public/field_editor/components/field_format_editor/index.js

```diff
@@ -18,3 +18,4 @@
  */
 
 export { FieldFormatEditor } from './field_format_editor';
+export { editors } from './get_editors';
```

Comment:
This export is one way to access the same editors in Lens and Management. This is no longer the desired UX for Lens, so I will probably remove this.